### ISSUE TITLE
Empty options cause "Cannot read property 'specs' of undefined" in Codesandbox

### ISSUE THREAD

**Issue by bodazhao** [bartlomiejzuber/use-open-window#188]
If you leave the `options` parameter empty, it will cause error 

```tsx
import { useOpenInWindow } from "use-open-window";

const url = "https://www.youtube.com/";

export default function App() {
  const [handleWindowOpen] = useOpenInWindow(url);

  return (
    <div className="App">
      <h1>Hello CodeSandbox</h1>
      <h2>Edit to see some magic happen!</h2>
      <button onClick={handleWindowOpen}>Click YouTube</button>
    </div>
  );
}
```

Error in Codesandbox

![image](https://user-images.githubusercontent.com/6238558/122052612-c3b7c800-cddd-11eb-9672-2d989ac901f1.png)

Although we can easily bypass this with an empty object and it will fix the errror

```tsx
const [handleWindowOpen] = useOpenInWindow(url, {}); // not ideal syntax
```

**Comment by bartlomiejzuber** [comment]
Hi @bodazhao , first of all thank you for posting this.
I've checked it and found out that the issue you're posting here was fixed in previous deploy although file built version has not been updated with newer one so in fact it was an issue with last deployment.

Anyway it's now fixed in version **1.3.4**.
https://codesandbox.io/s/cocky-flower-7m9my?file=/src/App.js

Thanks!

**Comment by bodazhao** [comment]
awesome, cheers for the quick reply!
